### PR TITLE
Feat - Fetching data from api

### PR DESCRIPTION
**Pull request by Mohasin-Haque**

### CODE REVIEW

**Review by chanchal-panpaliya — APPROVED** [review]
code looks good

File: src/backend/db/products.js [hunk 1]
```diff
@@ -8,23 +10,50 @@ import { v4 as uuid } from "uuid";
...
   },
   {
     _id: uuid(),
-    title: "Think and Grow Rich",
-    author: "Shiv Khera",
-    price: "1000",
-    categoryName: "fiction",
+    image: saree6,
+    title: "Maroom Banarasi Saree",
+    // author: "Shiv Khera",
```
Comment (akash-1318) [inline src/backend/db/products.js]: In case you forgot just remove the commented code

File: src/pages/products-page/ProductsPage.jsx [hunk 2]
```diff
@@ -1,7 +1,27 @@
...
 import "./products-page.css";
 
 const ProductsPage = () => {
+     
+    useEffect(()=>{
+        fetchProducts()
+    }, []);
+
+    const [loader, setLoader] = useState(false);
+    const [products, setProducts] = useState([]);
```
Comment (akash-1318) [inline src/pages/products-page/ProductsPage.jsx:9]: Just a suggestion - when you create states just create them on top of everything inside functional component.

**Review by akash-1318 — COMMENTED** [review]
Just added some suggestion besides that everything looks good with proper file structure.

File: src/pages/products-page/ProductsPage.jsx (hunk 2 above)
Comment (Mohasin-Haque) [reply src/pages/products-page/ProductsPage.jsx:9]: done

File: src/backend/db/products.js (hunk 1 above)
Comment (Mohasin-Haque) [reply src/backend/db/products.js]: done

**Pull request merged**